source/2020-08-25-accessibility-working-group-update.md

```diff
@@ -0,0 +1,73 @@
+
+To add another layer of support and enhance the overall developer experience, the team also decided that it would be useful to have an interactive way to create new Ember apps. After some discussion, [Melanie Sumner](https://github.com/melsumner) authored [RFC 638](https://emberjs.github.io/rfcs/0638-interactive-app-creation.html), and that RFC was accepted and merged on August 10th. Implementation is still pending.
+
+Additional support came through [updating the Ember Guides](https://guides.emberjs.com/release/accessibility/application-considerations/#toc_lang-attribute), with both prose and art, thanks to team members [Ava Wroten](https://github.com/hergaiety) and [Lenora Porter](https://github.com/lenoraporter). [Melanie Sumner](https://github.com/melsumner) also [added a new rule](https://github.com/ember-template-lint/ember-template-lint/pull/1386) to the [Ember Template Lint addon](https://github.com/ember-template-lint/ember-template-lint) called `require-lang`, which checks to make sure that the `lang` attribute is present on the `<html>` attribute. This rule will be effective in v3.0 of the addon, thanks to [PR 1232](https://github.com/ember-template-lint/ember-template-lint/pull/1232) by [lifeart](https://github.com/lifeart), which adds the ability to lint `.html` files.   
+
+### Label and Input Support
+
+There were three things the team decided to do in this area: update the guides to provide more explicit guidance, provide an addon that would help provide `id`s within a specific context, and write an RFC to propose a more permanent solution. 
+
+[Benjamin Jegard](https://github.com/kamikillerto) created the [Ember Context Id Helper addon](https://github.com/KamiKillertO/ember-context-id-helper). The `{{context-id}}` generates a unique `id` for a given context. Because it uses the ember guidFor function under the hood, it will always return the same `id` for the same context. A use case for `context-id` helper is to programmatically associate `<label>`s and `<input>` element using the `<label>` `for` attribute and the `<input>`s `id` attribute. 
+
+This addon took inspiration from the pre-RFC for the (guid-for) helper, which resulted in an [RFC for a new built-in template helper](https://github.com/steveszc/rfcs/blob/id-helper/text/0000-id-helper.md), authored by [Steve Szczecina](https://github.com/steveszc). 
```

Comment:
I think the parentheses came from Steve's use of them in the [pre-RFC](https://github.com/emberjs/rfcs/issues/612)? I think double curly braces may better indicate a helper.

```suggestion
This addon took inspiration from the pre-RFC for the `{{guid-for}}` helper, which resulted in an [RFC for a new built-in template helper](https://github.com/steveszc/rfcs/blob/id-helper/text/0000-id-helper.md), authored by [Steve Szczecina](https://github.com/steveszc). 
```